src/llm/apis/openai_completions.cpp

```diff
@@ -613,7 +613,7 @@ std::string OpenAIChatCompletionsHandler::serializeUnaryResponse(const std::vect
         writer.Int(index++);
         // logprobs: object/null; Log probability information for the choice. TODO
         writer.String("logprobs");
-        if (this->request.logprobschat || this->request.logprobs > 0) {
+        if (this->request.logprobschat || this->request.logprobs) {
```

Comment:
is logprobs signed or unsigned int? if its signed we change functionality here (might be <0 which now if will be true)

Reply:
but logprobs is bool, right?